util/requirements.txt

```diff
@@ -0,0 +1,4 @@
+certifi==2017.11.5
```

Comment:
N.B., should be added to `Dockerfile.dev` if #214 lands before this PR.

Reply:
As discussed elsewhere, probably best to port Docker to pull from known requirements.txt files for `pip install -r` as it re-uses behaviour that people know/expect.